.vscode/settings.json

```diff
@@ -13,5 +13,67 @@
     "**/resources/_gen": true,
     ".hugo_build.lock": true
   },
-  "debug.internalConsoleOptions": "neverOpen"
+  "debug.internalConsoleOptions": "neverOpen",
+  "cSpell.words": [
```

Comment:
Not a fan of this...like half of the keywords are listed here...and I'm sure this list will just ever grow. For now, let's remove it.